charts/tensorleap/Chart.lock

```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: ingress-nginx
+  repository: https://kubernetes.github.io/ingress-nginx
+  version: 4.7.0
+- name: datadog
+  repository: https://helm.datadoghq.com
+  version: 3.59.2
+digest: sha256:f48835ab44b0bbbdd6ae32b84776732e993a5ba1aa8824469aff16060f2df2f5
+generated: "2024-03-25T09:11:17.054729241Z"
```

Comment:
remove the lock file - it causes problems on `helm release`

Reply:
Should it be git-ignored then?

Reply:
Yes, Right